ScoutSuite/output/data/inc-scoutsuite/pagination.js

```diff
@@ -0,0 +1,162 @@
+var defaultPageSize = 2
```

Comment:
Same comment as above.
```suggestion
const defaultPageSize = 2
```